### ISSUE TITLE
Preview to show EAI% rather than absolute EAI values?

### ISSUE THREAD

**Issue by ConnectedSystems** [GFDRR/rdl-pyTools#7]
When viewing the result preview the color scale may be misleading due to outliers.

Suggest using a logarithmic colormap instead:

With absolute values:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/4075136/153744777-4f44c421-150a-48c1-833b-3a1c9fd9726d.png">

With log transformed values:

<img width="409" alt="image" src="https://user-images.githubusercontent.com/4075136/153746214-77b7e448-0839-4bc0-a607-39ef4b2a4a06.png">

Although the color bar needs to be fixed up as the bin values at the lower end get squashed together:

<img width="230" alt="image" src="https://user-images.githubusercontent.com/4075136/153746228-a9cc83d6-a73e-40a4-8eae-288bdbad0a1f.png">

**Comment by matamadio** [comment]
I don't agree on showing EAI% as it could be very misleading; I verified that the numbers shown are correct, that ADM3 unit has much more people exposed than elsewhere; so it should pop out.
But I agree on better legend classification based on quantile to avoid flattening all other values.

Could you please increase height of the output line a bit? Scrollbar is there but yeah:

![immagine](https://user-images.githubusercontent.com/44863827/153929175-1d5f673e-406d-4e70-99bd-6be44bfe42a7.png)

Note also how dropdowns squash the labels:
![immagine](https://user-images.githubusercontent.com/44863827/153929095-c4baeb06-549d-4b17-b321-3f6ab7c19bbf.png)

**Comment by ConnectedSystems** [comment]
> I don't agree on showing EAI% as it could be very misleading; I verified that the numbers shown are correct, that one ADM unit has much more people exposed than elsewhere; so it should pop out.

Yes sorry, I realised this while typing it up but forgot to update the issue title. This issue is on using a non-linear color scale.

Could you click on the cell (blank area to the left) to expand the display? That should solve your first issue.
I'll look into the second soon.

**Comment by matamadio** [comment]
It seems it is something that can be somehow improved by using normalisation:

```
from matplotlib import colors
divnorm=colors.TwoSlopeNorm(vmin=-5., vcenter=0., vmax=10)
pcolormesh(your_data, cmap="coolwarm", norm=divnorm)
```

https://stackoverflow.com/questions/7404116/defining-the-midpoint-of-a-colormap-in-matplotlib
